src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4317,8 +4317,8 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="SubmissionCanHaveAtMostOne" xml:space="preserve">
     <value>Submission can have at most one syntax tree.</value>
   </data>
-  <data name="SyntaxTreeNotFoundTo" xml:space="preserve">
-    <value>SyntaxTree '{0}' not found to remove</value>
+  <data name="SyntaxTreeNotFoundToRemove" xml:space="preserve">
```

Comment:
Please find at least one test affected by each of the resource string changes and update the expectations.
(To do that, you run the test with an empty `VerifyDiagnostics()` and the test error will give you the text to copy/paste to update the baseline)

Reply:
Those resource strings are exception messages, not diagnostics.

Should I test the messages by e.g. changing [this test](https://github.com/dotnet/roslyn/blob/88801fa/src/Compilers/CSharp/Test/Symbol/Compilation/CompilationAPITests.cs#L1325-L1334) to the following?

```c#
AssertEx.Throws<ArgumentException>(
    () => comp.ReplaceSyntaxTree(newTree: SyntaxFactory.ParseSyntaxTree("Using System;"), oldTree: t1),
    e => Assert.Equal(CSharpResources.SyntaxTreeNotFoundToRemove, e.Message));
```

Doing that doesn't really seem worth it to me.

Or am I missing something and there is a way to test exception messages using `VerifyDiagnostics`?

Reply:
I think it's fine to leave as-is, given that it's for exceptions not diagnostics.